### ISSUE TITLE
Freetype doesn't copy it's dll

### ISSUE THREAD

**Issue by tntxtnt** [Microsoft/vcpkg#5349]
Build log for `dbg`
```
...
CMakeFiles\freetype.dir\src\base\ftver.rc.res  /out:freetyped.dll /implib:freetyped.lib /pdb:freetyped.pdb /dll /version:6.16 /machine:x64 /debug /INCREMENTAL  H:\vcpkg\installed\x64-windows\debug\lib\zlibd.lib H:\vcpkg\installed\x64-windows\debug\lib\bz2d.lib H:\vcpkg\installed\x64-windows\debug\lib\lH:\vcpkg\installed\x64-windows\debug\lib\libpng16d.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib  && cd ."
Microsoft (R) Incremental Linker Version 14.16.27026.1
Copyright (C) Microsoft Corporation.  All rights reserved.

   Creating library freetyped.lib and object freetyped.exp
Microsoft (R) Incremental Linker Version 14.16.27026.1
Copyright (C) Microsoft Corporation.  All rights reserved.

[42/43] cmd.exe /C "cd /D H:\vcpkg\buildtrees\freetype\x64-windows-dbg && H:\vcpkg\downloads\tools\cmake-3.12.4-windows\cmake-3.12.4-win32-x86\bin\cmake.exe -P cmake_install.cmake"
-- Install configuration: "Debug"
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/include/freetype2
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/include/freetype2/freetype
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/include/freetype2/freetype/config
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/include/freetype2/freetype/config/ftconfig.h
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/include/freetype2/freetype/config/ftheader.h
...
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/include/freetype2/ft2build.h
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/include/freetype2/freetype/config/ftoption.h
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/lib/freetyped.lib
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/share/freetype/freetype-config.cmake
-- Installing: H:/vcpkg/packages/freetype_x64-windows/debug/share/freetype/freetype-config-debug.cmake
ibpng16d.lib H:\vcpkg\installed\x64-windows\debug\lib\zlibd.lib H:\vcpkg\installed\x64-windows\debug\lib\bz2d.lib 
```

and `rel`:
```
...
CMakeFiles\freetype.dir\src\base\ftver.rc.res  /out:freetype.dll /implib:freetype.lib /pdb:freetype.pdb /dll /version:6.16 /machine:x64 /DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF  H:\vcpkg\installed\x64-windows\lib\zlib.lib H:\vcpkg\installed\x64-windows\lib\bz2.lib H:\vcpkg\installed\x64-windows\lib\libpng16.lib H:\vcpkg\installed\x64-windows\lib\zlib.lib H:\vcpkg\installed\x64-windows\lib\bz2.lib H:\vcpkg\installed\x64-windows\lib\libpng16.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib  && cd ."
Microsoft (R) Incremental Linker Version 14.16.27026.1
Copyright (C) Microsoft Corporation.  All rights reserved.

   Creating library freetype.lib and object freetype.exp
[42/43] cmd.exe /C "cd /D H:\vcpkg\buildtrees\freetype\x64-windows-rel && H:\vcpkg\downloads\tools\cmake-3.12.4-windows\cmake-3.12.4-win32-x86\bin\cmake.exe -P cmake_install.cmake"
-- Install configuration: "Release"
-- Installing: H:/vcpkg/packages/freetype_x64-windows/include/freetype2
-- Installing: H:/vcpkg/packages/freetype_x64-windows/include/freetype2/freetype
-- Installing: H:/vcpkg/packages/freetype_x64-windows/include/freetype2/freetype/config
-- Installing: H:/vcpkg/packages/freetype_x64-windows/include/freetype2/freetype/config/ftconfig.h
-- Installing: H:/vcpkg/packages/freetype_x64-windows/include/freetype2/freetype/config/ftheader.h
...
-- Installing: H:/vcpkg/packages/freetype_x64-windows/include/freetype2/ft2build.h
-- Installing: H:/vcpkg/packages/freetype_x64-windows/include/freetype2/freetype/config/ftoption.h
-- Installing: H:/vcpkg/packages/freetype_x64-windows/lib/freetype.lib
-- Installing: H:/vcpkg/packages/freetype_x64-windows/share/freetype/freetype-config.cmake
-- Installing: H:/vcpkg/packages/freetype_x64-windows/share/freetype/freetype-config-release.cmake
```

It builds both `freetype.dll` and `freetyped.dll` but doesn't install them.

**Comment by Voskrese** [comment]
edit port file in copy *.dll to install ...

**Comment by PhoebeHui** [comment]
@TheScarfix, @vicroms, it looks related to https://github.com/Microsoft/vcpkg/commit/60deab0a29b64a739f39d60887f3b3d556afcc0e, freetype.dll export in previous version.

**Comment by TheScarfix** [comment]
got a fix but vcpkg won't apply the patch and I can't figure out why

**Comment by tntxtnt** [comment]
Fixed by #5354
